game/src/sandbox/dashboards/travel_times.rs

```diff
@@ -455,10 +486,14 @@ fn contingency_table(ctx: &mut EventCtx, app: &App, filter: &Filter) -> Widget {
 
     Widget::col(vec![
         Text::from_multiline(vec![
-            Line("Time difference by trip length").small_heading(),
-            Line("Grouped by the length of the trip before your changes."),
+            Line("Aggregate difference by trip length").small_heading(),
```

Comment:
Endlessly tweaking copy... better or worse?

Reply:
I like the explanatory text below. The two titles feel about the same to me. If I think about it carefully, "trip length" is kind of misleading; we mean duration, not distance. But not sure that's important to call out since "duration" sounds awkward. The axis labels and tooltips make it clear enough